docs/user-guide/install.md

```diff
@@ -158,3 +158,40 @@ Windows service vdvs uninstalled successfully!
 ```
 PS C:\> Set-ExecutionPolicy Unrestricted -Scope Process -Force
 ```
+
+## Upgrade Instructions
+
+The volume driver can be upgraded or reinstalled any time with no impact to the user data. User data in volumes existing before an upgrade of the volume driver is retained as it is and is accessible post an upgrade.
+
+Our volume managed plugin is taking care of the control path (management layer) and doesn't deal with data path. So during the upgrade process, only management related operations will be affected and data will be intact. Management operations like create/attach/detach/delete volume using driver may run into an error for the time being until the upgrade process is finished.
```

Comment:
"The volume plugin ....

Reply:
"Management operations like create/attach/detach/delete volume using driver may run into an error for the time being ..." Could be like

"Management operations like create/attach/detach/delete volume may fail until ......"